### PR TITLE
feature/fix-no-repository-found: 

### PR DESCRIPTION
**Pull request by tgrrr**
moved error message if there is no repository loaded to vscode console

### CODE REVIEW

**Comment by tgrrr** [comment]
Fixes error message on VS Code:

> Error: warning: Not a git repository. Use --no-index to compare two paths outside a working tree usage: git diff --no-index [<options>] <path> <path> Diff output format options -p, --patch generate patch -s, --no-patch suppress diff output -u generate patch -U, --unified[=<n>] generate diffs with <n> lines context -W, --function-context generate diffs with <n> lines context --raw generate the diff in raw format --patch-with-raw synonym for '-p --raw' --patch-with-stat synonym for '-p --stat' --numstat machine friendly --stat --shortstat output only the last line of --stat -X, --dirstat[=<param1,param2>...] output the distribution of relative amount of changes for each sub-directory --cumulative synonym for --dirstat=cumulative --dirstat-by-file[=<param1,param2>...] ...

File: src/extension.ts [hunk 1]
```diff
@@ -2,6 +2,11 @@ import * as vscode from "vscode";
 import SimpleGit from "simple-git/promise";
 import Yukikaze from "yukikaze";
 
+interface errorObj {
```
Comment (Pepijn98) [inline src/extension.ts:5]: Can you rename the interface to `GitError`, looks and sounds a lot better.

File: src/extension.ts [hunk 2]
```diff
@@ -10,7 +15,12 @@ const interval = new Yukikaze();
 export function activate(context: vscode.ExtensionContext) {
 	const config = vscode.workspace.getConfiguration();
 	const minutes = config.get<number>("commitReminder.interval") ?? 30;
-	startInterval(minutes);
+
+    git.checkIsRepo()
+        .then((isRepo: boolean) => isRepo && startInterval(minutes))
+        .catch((error: errorObj) => {
```
Comment (Pepijn98) [inline src/extension.ts:21]: Same here `GitError`

File: src/extension.ts [hunk 3]
```diff
@@ -10,7 +15,12 @@ const interval = new Yukikaze();
...
+            console.error('vscode-commit-reminder - no git repository found: ', error.name, ' ', error.message);
```
Comment (Pepijn98) [inline src/extension.ts:22]: Can you use double quotes here instead of single quotes

**Review by Pepijn98 — CHANGES_REQUESTED** [review]
Looks good besides the minor visual "errors", if those are fixed I'll merge it

**Pull request merged**